src/components/landing/listing.js

```diff
@@ -3,22 +3,42 @@ import Image from "next/image";
 import styles from "./listing.module.css";
 
 function Listing({ data }) {
-  const avatarWidth = 50
-  const avatarHeight = 50
+  const avatarWidth = 50;
+  const avatarHeight = 50;
   return (
     <>
       <article className={styles.listing}>
         <div className={styles.listing__image__container}>
           <Image
             src="/image-placeholder.png"
-            width={avatarWidth} height={avatarHeight}
-            alt={`${data.community_name} avatar`}  />
+            width={avatarWidth}
+            height={avatarHeight}
+            alt={`${data.community_name} avatar`}
+          />
         </div>
         <div className={styles.listing__content}>
-          <h2 className={styles.listing__content__name}>{data.community_name}</h2>
-          <p className={styles.listing__content__manager}> {data.community_manager} </p>
-          <p className={styles.listing__content__description}> {data.community_description}</p>
-          <Link href={`/${encodeURIComponent(data.communitd_id)}`}  className={styles.listing__view__more}>view more
+          <h2 className={styles.listing__content__name}>
+            {data.community_name}
+          </h2>
+          <p className={styles.listing__content__manager}>
+            {" "}
+            {data.community_manager}{" "}
+          </p>
+          <p className={styles.listing__content__description}>
+            {" "}
+            {data.community_description}
+          </p>
+          <Link
+            href={{
+              pathname: "/[community_id]",
+              query: {
+                community_id: data.community_id,
+                title: data.uid,
```

Comment:
Why are we using `data.uid` for the `title` here? is this the `title` of the web page? Or something else?